samples/extensions/open_cl_interop/open_cl_interop.cpp

```diff
@@ -620,12 +620,12 @@ void OpenCLInterop::prepare_shared_image()
 	mem_properties.push_back((cl_mem_properties) handle);
 #else
 	int fd = get_vulkan_memory_handle(shared_image.memory);
-	mem_properties.push_back((cl_mem_properties) CL_EXTERNAL_MEMORY_HANDLE_OPAQUE_FD_KHR);
```

Comment:
Same as above. Since I wrote that code I'm curious to know what type of cast is better and which one we should use in the future.

Reply:
Thank you very much. Good to know :)